esxi/assets/configuration/spec.yaml

```diff
@@ -30,4 +30,16 @@ files:
           value:
             type: string
             example: <PASSWORD>
+        - name: use_guest_hostname
+          description: |
+            If true, the check will use the guest hostname for VMs instead of the VM name
+            This requires the VM to have VMware tools installed in it. If the guest hostname is
+            not available, it will fallback to the VM name.
+            Use this if you install the agent in VMs so that the hosts are not duplicated
+            in the web application UI.
```

Comment:
```suggestion
            If true, the check will use the guest hostname for VMs instead of the VM name.
            This requires the VM to have VMware tools installed. If the guest hostname is
            not available, it will fallback to the VM name.
            Use this if you install the agent on VMs to avoid duplicate entries for hosts
            in the web application UI.
```